_sass/_layout.scss

```diff
@@ -2,11 +2,6 @@
  * Site header
  */
 .site-header {
-    background: linear-gradient(to bottom, white 70%, $scheme-color-1);
-    @include media-query($on-palm) {
-        background: white;
-    }
-
     border-top: 5px solid $grey-color-dark;
     border-bottom: 1px solid $grey-color-light;
     min-height: 56px;
```

Comment:
I think this three lines could be removed too:
- the two first to remove the useless horizontal lines in the header
- the third one, because the height is guaranteed by the logo image and "line-height: 56px;" elsewhere.

Actually, I think "line-height: 56px" is too big for the menu items. Maybe something slightly larger than the font-size like "line-height: 32px;" would be enough